types/shared.d.ts

```diff
@@ -1,24 +1,36 @@
-declare module "skedify-sdk" {
+declare module 'skedify-sdk' {
   export interface APIError<Entity> {
-    code: string;
-    message: string;
-    ticket_id?: string;
+    code: string
+    message: string
+    ticket_id?: string
     fields?: {
-      [Key in keyof Entity]?: string[];
-    };
+      [Key in keyof Entity]?: string[]
+    }
   }
 
   export interface APIResponse<Entity> {
-    data: Entity;
-    warnings: unknown[];
-    errors: APIError<Entity>[];
-    status: number;
+    data: Entity
+    warnings: unknown[]
+    errors: APIError<Entity>[]
+    status: number
+    paging: PagingResponse
```

Comment:
Paging metadata will be optional as far as I know?

```suggestion
    paging?: PagingResponse
```